simple_imports/README.md

```diff
@@ -0,0 +1,41 @@
+# Simple Imports
+
+This folder contains configs for imports using [Simple Stats
+Importer](https://github.com/datacommonsorg/import/edit/master/simple/README.md).
+
+## Import config
+
+Each import has a sub-folder with a config file `<import-name>/simple_import.json`.
+The parameters for the simple import config file are described in detail 
+[here](https://github.com/datacommonsorg/import/blob/master/simple/stats/config.md).
+
+The config has the following parameters for automated updates:
+```
+{
+  "dataDownloadUrl": ["<GCS path containing input files>"],
+  "inputFiles": {
+    "<file1.csv>": {
+      "importType": "observations",
+      "entityType": "Country",
+    },
+  },
+  "provenanceUrl": "<url describing the import source data>",
+  "provenanceDescription": "<description including citation for the import source>",
+  "cronSchedule": "<Minute> <Hour> <Day-of-month> <Month> <Day-of-week>"
+}
+```
+
+Imports with a `cronSchedule` are processed periodically as a cloud run job by
+the [import automation](https://github.com/datacommonsorg/data/tree/master/import-automation).
+
+
+## Adding a new import
+
+To add a new data source:
+1. Copy the input files to a GCS bucket that is
+read accessible to the service account `965988403328-compute@developer.gserviceaccount.com`.
+2. Create a sub-folder for the import under `data/simple_imports/<import_name>`.
+3. Add an import config `simple_import.json` with the list of input files to be
```

Comment:
Is it `simple_import.json` or `import_config.json`?

Reply:
changed to import_config.json